charts/sda-svc/values.yaml

```
@@ -239,7 +239,9 @@ global:
       iss:
         - iss: "https://login.elixir-czech.org/oidc"
           jku: "https://login.elixir-czech.org/oidc/jwk"
-    serveUnencryptedData: false
+    serveDecrypted:
+      c4ghKeyFile: ""
+      secretName: ""
```

```suggestion
    serveDecrypted:
      c4ghKeyFile: ""
      passphrase: ""
      secretName: ""
```

Private key needs a passphrase.

---

The passphrase is intended to be read from the secret, similarly as to how we have it in `ingest-deploy.yaml`, see [here](https://github.com/neicnordic/sensitive-data-archive/blob/1176989fe22f88b526cc9603d861cf112ffdec7e/charts/sda-svc/templates/ingest-deploy.yaml#L212). 
Imo this is better than passing the secret via a `values.yaml` file.

I can either add a note in the README saying that the secretName should include the passphrase
or add the `passphrase` variable as suggested above.

---

Right, in the notes it goes then.